server/graphql/v2/mutation/HostApplicationMutations.ts

```diff
@@ -218,14 +218,15 @@ const sendPrivateMessage = async (host, collective, message: string): Promise<vo
   const adminUsers = await collective.getAdminUsers();
   await emailLib.send(
     'host.application.contact',
-    NO_REPLY_EMAIL,
+    [],
```

Comment:
I think this is an bug where we put the `NO_REPLY_EMIL` on the recipient field that doesn't make sense. 🤔

Reply:
This was done because `generateEmailFromTemplateAndSend` expects a `recipient` to be set, but passing an empty array indeed seems like a better hack :+1: